CardPayments/src/main/java/com/paypal/android/cardpayments/CardClient.kt

```diff
@@ -129,24 +117,23 @@ class CardClient internal constructor(
         if (browserSwitchResult != null && approveOrderListener != null) {
             approveOrderListener?.onApproveOrderThreeDSecureDidFinish()
             when (browserSwitchResult.status) {
-                BrowserSwitchStatus.SUCCESS -> getOrderInfo(browserSwitchResult)
+                BrowserSwitchStatus.SUCCESS -> handleBrowserSwitchSuccess(browserSwitchResult)
                 BrowserSwitchStatus.CANCELED -> notifyApproveOrderCanceled()
             }
         }
     }
 
-    private fun getOrderInfo(browserSwitchResult: BrowserSwitchResult) {
+    private fun handleBrowserSwitchSuccess(browserSwitchResult: BrowserSwitchResult) {
         ApproveOrderMetadata.fromJSON(browserSwitchResult.requestMetadata)?.let { metadata ->
             CoroutineScope(dispatcher).launch(exceptionHandler) {
                 try {
-                    val getOrderResponse = checkoutOrdersAPI.getOrderInfo(GetOrderRequest(metadata.orderID))
                     analyticsService.sendAnalyticsEvent(
                         "card-payments:3ds:get-order-info:succeeded",
                         metadata.orderID
                     )
                     val deepLinkUrl = browserSwitchResult.deepLinkUrl
-                    val result = getOrderResponse.run {
-                        CardResult(orderId, orderStatus, paymentSource, deepLinkUrl)
+                    val result = metadata.run {
+                        CardResult(orderID, null, paymentSource, deepLinkUrl)
```

Comment:
Can we leave a TODO to address this always sending `null`? We'll have to align the result type on both iOS and Android

Reply:
This is a good call we should remove that property altogether. I'll remove.

Reply:
Hm, I'd vote to remove it over sending "Unknown"